src/integrations/elixir.html

```diff
@@ -0,0 +1,37 @@
+{{#> layout page="integrations_details_intro"}}
+
+<!-- prettier-ignore -->
+{{> integrations_details_intro 
+  
+      heading="Use Elixir directly in your notebook"  
+      description="Livebook has an interactive environment to code in Elixir. You can use it for learning Elixir, exploring Elixir packages, prototyping, scripts, interactive data apps and more." 
+      image_src="/images/integrations_details/elixir_details.svg"
+
+}}
+
+{{> integrations_features
```

Comment:
@paulo-valim I think that should actually be `integrations_details_features`, right? Because that's the name of the partial.

When I ran that PR locally, I got the following error:

![CleanShot 2023-01-04 at 13 41 02](https://user-images.githubusercontent.com/2719/210605131-ba9a721a-6925-49f9-9617-561a504bc0db.png)